### PR TITLE
Added login bar and updated nav bar menu options, closes #107

### PR DESCRIPTION
**Pull request by nathanows**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -21,7 +21,11 @@ def redirect_after_login
     elsif session[:return_to]
       redirect_to session[:return_to]
     else
-      OrdersController.create
+      @order = Order.create(user_id: current_user.id, status: "ordered")
+      @order.create_order_items(@cart)
+      @order.update_attributes(total_price: @order.order_total)
+      @cart.clear
+      redirect_to order_path(@order)
```
Comment (scottcrawford03) [inline app/controllers/application_controller.rb:28]: is this the start of the cart logic? :kr:

Comment (KristaANelson) [inline app/controllers/application_controller.rb:28]: yes, this allows a user not to be redirected back to the checkout page if that is how they are prompted to login. This works now, but will add to the waffle list to add a checkout summary page which will be a much cleaner way to handle this.

File: app/views/shared/_nav_bar.html.erb [hunk 2]
```diff
@@ -28,16 +45,12 @@
...
           <%= menu_item "Dashboard", "#" %>
         <% end %>
-        <%= drop_down_divider %>
-        <%= menu_item "Logout", logout_path, method: :delete %>
       <% end %>
-    <% else %>
-      <%= menu_item "Login", login_path %>
     <% end %>
   <% end %>
-<% end %>
```
Comment (KristaANelson) [inline app/views/shared/_nav_bar.html.erb:43]: Pulled this down to take a look, and it looks awesome!!! :sparkles: Sparkles all around!! :sparkles:

**Comment by KristaANelson** [comment]
Looks good! We have some refactoring we can do, but tests are passing and much improvement! :purple_heart:

**Pull request merged**